### PR TITLE
adds "download" to tool names

### PR DESCRIPTION
**Pull request by bwlang**
I initially expected the tools to take a .sra file as input

### CODE REVIEW

**Comment by bwlang** [comment]
@bgruening : PR as requested ;)

File: tools/sra-tools/sra_pileup.xml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-<tool id="sra_pileup" name="Generate pileup format" version="@VERSION@.2">
+<<<<<<< HEAD
```
Comment (shiltemann) [inline tools/sra-tools/sra_pileup.xml]: oops?

Comment (shiltemann) [inline tools/sra-tools/sra_pileup.xml]: haha you fixed it as I hit enter, disregard :P

File: tools/sra-tools/sra_pileup.xml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<tool id="sra_pileup" name="Generate pileup format" version="@VERSION@.2">
+<tool id="sra_pileup" name="Download and Generate Pileup Format" version="@VERSION@.2">
     <description>from NCBI sra</description>
```
Comment (nsoranzo) [inline tools/sra-tools/sra_pileup.xml]: sra -> SRA

File: tools/sra-tools/fastq_dump.xml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<tool id="fastq_dump" name="Extract reads in Fastq/a" version="@VERSION@.3">
+<tool id="fastq_dump" name="Download and Extract Reads in Fastq/a" version="@VERSION@.3">
```
Comment (nsoranzo) [inline tools/sra-tools/fastq_dump.xml]: Fastq/a -> FASTQ/A

**Comment by bgruening** [comment]
Thanks a bunch @bwlang! If you can fix @nsoranzo comments we can merge!

**Comment by bgruening** [comment]
Thanks.

**Pull request merged**